client/src/App.js

```diff
@@ -21,6 +23,7 @@ class App extends Component {
           <h2>Welcome to React</h2>
           {this.state.text}
         </div>
+        <RecommendedCategories />
```

Comment:
Can you move this under TopCharts. I think in the mock RecommendedCategories is below TopCharts.